iceoryx_dds/include/iceoryx_dds/dds/cyclone_data_writer.hpp

```diff
@@ -32,21 +32,53 @@ namespace dds
 class CycloneDataWriter : public iox::dds::DataWriter
 {
   public:
+    /// @brief delete default ctor
```

Comment:
Please do not document deleted methods and add an explanation on why they are deleted.

Reply:
@elfenpiff  for explanation should we use @note, something like this should be enough :
 
         /// @note delete copy and move ctor and assignment since they are not needed